logutil/logutil.go

```diff
@@ -76,3 +76,42 @@ func NewServerLogger(debug bool) log.Logger {
 	}()
 	return &swapLogger
 }
+
+// NewFmtLogger creates a standard logger for Kolide CLI tools.
+// The logger will output leveled logs with a
+// "severity" field set to either "info" or "debug".
+// The acceptable level can be swapped by sending SIGUSR2 to the process.
+func NewFmtLogger(debug bool) log.Logger {
+	base := log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
+	base = log.With(base, "ts", log.DefaultTimestampUTC)
+	base = SetLevelKey(base, "severity")
+	base = level.NewInjector(base, level.InfoValue())
+
+	lev := level.AllowInfo()
+	if debug {
+		lev = level.AllowDebug()
+	}
+
+	base = log.With(base, "caller", log.Caller(6))
+
+	var swapLogger log.SwapLogger
```

Comment:
i imagine LogFmtLogger will be used for short running processes, like CLI commands. In that case, is swapLogger useful?

Reply:
What if the CLI process is hung in some infinite loop or something and you want to turn on debug logging to see what it keeps doing?